config/tls.go

```diff
@@ -0,0 +1,49 @@
+package config
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+)
+
+func TLS(certPath string, keyPath string, clientCAPath string) (*tls.Config, error) {
+
+	tlsConfig := &tls.Config{}
+
+	if certPath != "" && keyPath != "" {
+
+		certData, err := ioutil.ReadFile(certPath)
+		if err != nil {
+			return nil, fmt.Errorf("error: unable read certificate: %s", err)
+		}
+
+		keyData, err := ioutil.ReadFile(keyPath)
+		if err != nil {
+			return nil, fmt.Errorf("error: unable read key: %s", err)
+		}
+
+		cert, err := tls.X509KeyPair(certData, keyData)
```

Comment:
Maybe we can just use `tls.LoadX509KeyPair` instead?

Reply:
sure. I was trying something else, then went back to this, and did not switch back to the easy wrapper.